### ISSUE TITLE
Question about urls stored in KeePass

### ISSUE THREAD

**Issue by bcavalluzzi** [perfectapi/CKP#109]
It seems I must enter the proper url in my web browser before utilizing the plugin to autofill the username and password. I store those urls in KeePass rather than bookmarking them. Is there a way for me to access them on my chromebook to open the desired webpage and then login using the username and password the extension reads?

**Comment by DukeyToo** [comment]
No, the extension does not work that way, but it is a good idea I think.

**Comment by nly95504** [comment]
I agree this would be a good feature. Keepass desktop works that way as well, and I use it all the time
